content/uk/case-studies/_index.html

```diff
@@ -0,0 +1,13 @@
+---
+title: Case Studies
+title: Приклади використання
+linkTitle: Case Studies
+linkTitle: Приклади використання
+bigheader: Kubernetes User Case Studies
+bigheader: Приклади використання Kubernetes від користувачів.
+abstract: A collection of users running Kubernetes in production.
+abstract: Підбірка користувачів, що використовують Kubernetes для робочих навантажень.
```

Comment:
These are duplicated.

Reply:
Yaml after merge use only last value so all works fine.
But yes, it can commented